app/javascript/utilities/podcastPlayback.js

```diff
@@ -22,14 +22,13 @@
  * the problem of using a method before it's defined:
  */
 
-/* global ahoy, Runtime */
 /* eslint no-use-before-define: 0 */
 /* eslint no-param-reassign: 0 */
 
-var audioInitialized = false;
+import ahoy from 'ahoy.js';
 
-function initializePodcastPlayback() {
-  var deviceType = 'web';
+export function initializePodcastPlayback() {
```

Comment:
Now that this was moved to webpack bundle there's a bunch of small changes to ensure eslint passes (i.e. var -> let/const)